### DISCUSSION TITLE
Use rekalogika/file-filepond with AssetMapper

### DISCUSSION THREAD

**Discussion by szal1k** [rekalogika/file-src#54] [Q&A]
I will test file-filepond with AssetMapper, follow the instructions in the article: https://rekalogika.dev/file-bundle/file-upload-filepond

After run: composer require rekalogika/file-filepond and complete installation, refresh page and browser console return error:
'TypeError: Specifier 'filepond' was itself a specifier, but was not bound to anything. Relative module specifiers must start with './', '../', or '/'.'

That means no install "filepond" library, i was checked and install bundle does not install library. After that i run:
php bin/console importmap:require filepond

After that the error is with 'filepond-plugin-file-encode':
'TypeError: Specifier 'filepond-plugin-file-encode' was itself a specifier, but was not bound to anything. Relative module specifiers must start with './', '../', or '/'.'

Can i must install all import file from 'file-filepond/assets/dist/filepond.js' manually or am I doing something wrong? I don't have experient with AssetMapper (always use webpack).

**Comment by priyadi** [comment]
I confess that part is relatively untested. I just tested and got the same error. We have a deployment with that configuration that works, but probably there are other things going on. The project I'm doing is using webpack encore, so at least in that configuration I can tell that it should work.

Allow me a couple of weeks to modernize to Symfony 7 standard.

**Comment by priyadi** [comment]
I went ahead and made sure everything works correctly with import maps. It should work without additional effort now.

I also created a demo on top of the official Symfony demo here: https://github.com/rekalogika/file-demo . See the last few commits to see how to do things with `rekalogika/file`.
